### PR TITLE
docs: Add object detection docs

### PR DESCRIPTION
**Pull request by chmjkb**
## Description
This PR adds useObjectDetection to the computer vision section of the docs

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (improves or adds clarity to existing documentation)

### Tested on
- [ ] iOS
- [ ] Android

### Checklist
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by NorbertKlockiewicz** [comment]
lgtm!

File: docs/docs/computer_vision/useObjectDetection.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  isModelReady: boolean;
+  isModelGenerating: boolean;
+  forward: (input: string) => Promise<Detection[]>;
+}
+```
+</details>
+
+### Running the model
+
+To run the model, you can use the `forward` method. It accepts one argument, which is the image. The image can be a remote URL, a local file URI, or a base64-encoded image. The function returns an array of `Detection` objects. Each object contains coordinates of the bounding box, the label of the detected object, and the confidence score. For more information, please refer to the reference or type definitions.
```
Comment (NorbertKlockiewicz) [inline docs/docs/computer_vision/useObjectDetection.mdx]: On my branch I added standalone page dedicated for loading models, you can link to it here.

Comment (chmjkb) [reply docs/docs/computer_vision/useObjectDetection.mdx]: Loading models? Or inferencing them?

Comment (NorbertKlockiewicz) [reply docs/docs/computer_vision/useObjectDetection.mdx]: There's a /fundamentals/loading-models page which includes 3 ways of loading model(https etc., require and local file), btw. I added comment in different line XD, I meant lines from 59 to 64

Comment (chmjkb) [reply docs/docs/computer_vision/useObjectDetection.mdx]: yeah, that's what i thought, i'll link to that :D

**Review by NorbertKlockiewicz — APPROVED** [review]

File: docs/docs/computer_vision/useObjectDetection.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+`useObjectDetection` is a hook that allows you to seamlessly integrate object detection into your React Native application. Currently, the SSDLite320Large model with a MobileNetv3 backbone is supported.
+
+## Reference
+```jsx
+import { useObjectDetection, SSDLITE_320_MOBILENET_V3_LARGE_URL } from 'react-native-executorch';
+
+function App() {
+  const ssdlite = useObjectDetection({
+    modelSource: SSDLITE_320_MOBILENET_V3_LARGE_URL, // alternatively, you can use require(...)
```
Comment (jakmro) [inline docs/docs/computer_vision/useObjectDetection.mdx]: Ultimately we opted for not using URL suffixes.

**Review by mkopcins — APPROVED** [review]

**Pull request merged**